small changes (#2)
opencasestudies/ocs-ripples-ancestry: index.Rmd

To cite this case study please use:

Kate Isaac, Hanako Osuga, Jeanne Chowning, Wright, Carrie (2025). https://github.com/opencasestudies/ocs-bp-co2-emissions. The Biology of Ancestry (Version v1.0.0).

####

> To access the GitHub repository for this case study see here: https://github.com/opencasestudies/ocs-ripples-ancestry.

Author: ```suggestion
To access the GitHub repository to see the source materials and data that went into making the case study, please see here: https://github.com/opencasestudies/ocs-ripples-ancestry.
```

To cite this case study please use:

Kate Isaac, Hanako Osuga, Jeanne Chowning, Wright, Carrie (2025). https://github.com/opencasestudies/ocs-bp-co2-emissions. The Biology of Ancestry (Version v1.0.0).

####

> In this space you should provide an open access link to the case study source files. This is necessary for users to get access to case study data, at a minimum. See the following example from the CO2 Emissions case study:

Author: ```suggestion
```

To access the GitHub repository for this case study see here: https://github.com/opencasestudies/ocs-ripples-ancestry.

> This case study is published as part of the [Reducing Inequities by Promoting Participatory Learning Experiences in Sciences (RIPPLES) Project](https://www.fredhutch.org/en/news/center-news/2024/09/science-education-partnership-ripples-grant.html) funded by R25GM154358, a Science Education Partnership Award (SEPA) from the National Institute of General Medical Sciences (NIGMS). The contents of this case study are solely the responsibility of the authors and do not represent the official views of the NIGMS or NIH.

Author: ```suggestion
This case study is published as part of the [Reducing Inequities by Promoting Participatory Learning Experiences in Sciences (RIPPLES) Project](https://www.fredhutch.org/en/news/center-news/2024/09/science-education-partnership-ripples-grant.html) funded by R25GM154358, a Science Education Partnership Award (SEPA) from the National Institute of General Medical Sciences (NIGMS). The contents of this case study are solely the responsibility of the authors and do not represent the official views of the Fred Hutch, NIGMS, or NIH.
```
